docs/development-guidelines/definition-of-done.md

```diff
@@ -0,0 +1,64 @@
+## Definition of done (DoD)
+
+The "Definition of Done" in a software project is like the finish line for each task or user story. It's a checklist that ensures everything related to a piece of work is completed to an acceptable standard defined by the acceptance criteria before considering it truly finished.
+
+In our case, we want template applications showcased in the gallery to guarantee developers can 
+
+- Discover
+- Setup
+- Adapt
+- Extend
+- Deploy and publish
+
+a functional application.
+
+## Acceptance criteria for gallery acceptance
+
+> [WARNING]
+> This is a draft
+
+The following checklist must be complete before a template is published
+
+# Repository Management
+
+- [] Standards compliant [README.md](../../README.md) as the one in the example, is in place
+- [] [License](../../LICENSE.md) is in place. Make sure you choose the [correct license](https://www.microsoft.com/en-us/legal/intellectualproperty/open-source)
+- [] [Security guidelines](../../SECURITY.md)are in place
+- [] [Contribution guidelines](../../CONTRIBUTING.md) are in place
+- [] [Code of conduct](.github/CODE_OF_CONDUCT.md) is in place
+- [] [Issue template](.github/ISSUE_TEMPLATE.md) is in place
+- [] Language, model and relevant technology topic labels are added, including `azd-templates` and `ai-azd-templates` (The latter is being created)
```

Comment:
Missing a comma after "model" in the list. https://learn.microsoft.com/en-us/style-guide/punctuation/commas

Reply:
Fixed.